### ISSUE TITLE
Any dataset-specific considerations for the mchesapeake or mcashewplant datasets?

### ISSUE THREAD

**Issue by yurujaja** [zhu-xlab/DOFA#22]
Hi,

I am trying to reproduce the semantic segmentation results in mycode base. While I could reproduce or achieve similar results to those reported for the nz-cattle, neon-tree, pv4ger-seg, and scarp datasets, I got quite large discrepancies  with the  m-cashewplant dataset and m-chesapeake dataset.

I was wondering if there are specific classes ignored during evaluation, as I noticed some classes labeled as “no data” or “background.” Alternatively, are there any dataset-specific considerations or factors that I might need to be aware of for these two datasets?

Thank you in advance for your help!

Best regards

**Comment by xiong-zhitong** [comment]
Hi, the discrepancies may come from the optimizer and hyperparameters: batch size, learning rate, and warmup strategy.

For these two datasets, the parameters we are using are:

```
    {
        "model": "dofa_seg",
        "dataset": "geobench_chesapeake",
        "task": "segmentation",
        "batch_size": 84,
        "epochs": 20,
        "lr": 0.005,
        "warmup_epochs": 3,
    },
    {
        "model": "dofa_seg",
        "dataset": "geobench_cashew",
        "task": "segmentation",
        "batch_size": 16,
        "epochs": 20,
        "lr": 0.005,
        "warmup_epochs": 3,
    },
```
BTW, we keep the batch size consistent for all the models. 
Feel free to let me know if you have any more questions.
